packages/common/src/api/developer-apps.ts

```diff
@@ -0,0 +1,80 @@
+import { z } from 'zod'
+
+import { ID } from 'models/Identifiers'
+import { createApi } from 'src/audius-query'
+import { encodeHashId } from 'utils/hashIds'
+
+const descriptionMaxLength = 28
```

Comment:
i think this should be 128 (assuming it's chars)

Reply:
oh yup great find